lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_release_notes.rb

```diff
@@ -4,11 +4,16 @@ class AndroidUpdateReleaseNotesAction < Action
         def self.run(params)
           UI.message "Updating the release notes..."
   
-          require_relative '../../helper/android/android_git_helper.rb'
           require_relative '../../helper/android/android_version_helper.rb'
+          require_relative '../../helper/release_notes_helper.rb'
+          require_relative '../../helper/git_helper.rb'
+
+          path = File.join(ENV["PROJECT_ROOT_FOLDER"] || '.', 'RELEASE-NOTES.txt')
           next_version = Fastlane::Helpers::AndroidVersionHelper.calc_next_release_short_version(params[:new_version])
-          Fastlane::Helpers::AndroidGitHelper.update_release_notes(next_version)
-            
+
+          Fastlane::Helper::ReleaseNotesHelper.add_new_section(path: path, section_title: next_version)
+          Fastlane::Helper::GitHelper.commit(message: "Update release notes", files: path, push: true)
```

Comment:
Not necessarily in scope for this change set, but, what do you think of

```suggestion
          Fastlane::Helper::GitHelper.commit(message: "Update release notes with #{next_version} section", files: path, push: true)
```

or

```suggestion
          Fastlane::Helper::GitHelper.commit(message: "Add empty #{next_version} section to release notes", files: path, push: true)
```

Or something else that adds the version number in the commit message for extra context.

Reply:
Good idea! Done in 310908ddad862919ded5418a493226ccc4bffd41